Framework/Crystal/src/FilterPeaks.cpp

```diff
@@ -84,6 +91,12 @@ void FilterPeaks::exec() {
     filterFunction = &HKL2;
   else if (FilterVariable == "Intensity")
     filterFunction = &intensity;
+  else if (FilterVariable == "Wavelength")
+    filterFunction = &wavelength;
+  else if (FilterVariable == "DSpacing")
```

Comment:
`FilterVariable` and `Operator` don't follow our naming conventions -  maybe these should be changed?